help_to_heat/settings.py

```diff
@@ -219,6 +221,31 @@ def show_toolbar(request):
     if EMAIL_BACKEND_TYPE not in ("FILE", "CONSOLE", "GOVUKNOTIFY"):
         raise Exception(f"Unknown EMAIL_BACKEND_TYPE of {EMAIL_BACKEND_TYPE}")
 
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "requests": {
+            "()": "django.utils.log.ServerFormatter",
+            "format": "[{server_time}] {message}",
+            "style": "{",
+        }
+    },
+    "handlers": {
+        "requests": {
+            "level": "INFO",
+            "class": "logging.StreamHandler",
+            "formatter": "requests",
+        },
+    },
+    "loggers": {
+        "requests": {
+            "handlers": ["requests"],
+            "level": "INFO",
+        },
+    },
+}
```

Comment:
Would be useful to know what each bit of this is doing and whether we need it all. For example we've got two `"level": "INFO"`s and we're also doing `requests_logger.info` in the middleware - do we need all of those?